docs/guide/content_rendering.md

```diff
@@ -979,11 +979,11 @@ Typical use cases include access to:
 - Main Location and alternative Locations for the current Content item
 - etc.
 
-There are three ways in which you can apply a custom controller:
+There are three ways in which you can apply a custom logic:
 
-- Configure a custom controller alongside regular matcher rules to use **both** your custom controller and the `ViewController` (recommended).
-- [**Override**](#overriding-the-built-in-viewcontroller) the built-in `ViewController` with the custom controller in a specific situation.
-- **Replace** the `ViewController` with the custom controller for the whole bundle.
+- Configure a custom controller alongside regular matcher rules to use **both** your custom controller and the `ViewController`.
```

Comment:
Naming wise, it would be a `custom view controller` or? /cc @bdunogier

Reply:
Yeah, it's a custom view controller.
But it doesn't run alongside with the `ViewController`. It replaces it. But since `ViewController::viewAction()` doesn't _do_ anything, it is perfectly regular.

Reply:
I have changed the name accordingly.